CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -1,46 +1,2 @@
-{% load static %}
-{% load inflection_extras %}
-{% for word in words %}
-<li class="search-results__result" data-cy="search-results">
-  <article class="definition box box--rounded" data-cy="search-result">
-    <header class="definition__header">
-      <h2 class="definition-title definition-title--search-result">
-
-        <dfn class="definition-title__word" data-cy="definition-title">
-          {# upon loaded, this <a> will be attached to onclick handler that loads #}
-          {# detailed lemma page. <a> is used for accessible styling #}
-          {% if word.is_lemma %}
-            <a class="definition-title__link" href="javascript:void(0)" data-lemma-id="{{ word.id }}">
-              {# data- attributes are supported in HTML5 not in XHTML #}
-              <span lang="cr">{{ word.text }}</span></a>
-          {% else %}
-              {# TODO: this should be the NORMATIZED matched inflection! #}
-              <span lang="cr">{{ word.text }}</span>
-          {% endif %}
-        </dfn>
-
-        <span class="definition-title__pos">({{ word|presentational_pos }})</span>
-      </h2>
-    </header>
-
-      {# Show the matched lemma (when this is NOT a lemma. #}
-      {% if not word.is_lemma %}
-      <p class="definition-title__link-to-lemma">
-        Form of
-        {# TODO: this link should go to the lemma search result, further down the page #}
-        <a href="#">{{ word.lemma.text }}</a>
-      </p>
-      {% endif %}
-
-    <ol class="meanings meanings--search-result">
-    {% for def in word.definitions.all %}
-      <li class="meanings__meaning" data-cy="lemma-meaning">{{ def.text }}
-        {% for source in def.source_ids %}
-          <cite class="cite-dict">{{ source }}</cite>
-        {% endfor %}
-      </li>
-    {% endfor %}
-    </ol>
-  </article>
-</li>
-{% endfor %}
+{% include 'CreeDictionary/cree-entries.html' %}
+{% include 'CreeDictionary/english-entries.html' %}
```

Comment:
So if I understand this correctly, first it shows Cree results, then it shows English results? That is kinda weird, but I won't block merge on it.